UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -192,8 +194,7 @@ class BTShopperInsightsClient_Tests: XCTestCase {
     }
 
     func testGetRecommendedPaymentMethods_withTokenizationKey_returnsError() async {
-        let apiClient = BTAPIClient(authorization: "sandbox_merchant_1234567890abc")!
-        let shopperInsightsClient = BTShopperInsightsClient(apiClient: apiClient)
+        let shopperInsightsClient = BTShopperInsightsClient(authorization: sandboxToken)
```

Comment:
Since this is only used in 1 test and invalid for this flow I think we can just hardcode it here.